recipes/Advection/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "iscd-advection" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}_{{ version }}.tar.gz
+  url: https://github.com/ISCDtoolbox/Advection/archive/refs/heads/master.zip
```

Comment:
Do not use a git branch (master) for a release. This zip file will change if more commits are added to the branch, so it is not an absolute point of reference. You should not create your own release version from this repository. Please ask the maintainers of these projects to create a tagged release for you.

Reply:
I totally agree about this. We had a business meeting with the maintainers (university researchers) and tried to explain why it was important to tag the software. They agreed  but... it might take some time. I could fork and tag myself but I suppose that it wouldn't be acceptable since it would not be the official source... Still, we need these dependencies for other packages so, could we go for the "master" package until there is an official tag ? Maybe I could try to get a zip package of a specific commit, would that be better ?

Reply:
I believe it is possible to download a tar/zip via commit hash from GitHub; that would be better. As for the version number of this package, you should choose something like `0.0.0rc0` where you would increment rc0, rc1, rc2 whenever you move to a new commit until they release something official.

Also, write a comment in the recipe explaining that there are no tagged releases which is why you are using archives of specific commits.

Reply:
I did as you suggested (thank you for the advice!). The code maintainer provided a few tags that were integrated, for the rest I fixed the 0.0.0rc0 and added the comment in the recipe